elixir/apps/web/test/web/live/resources/index_test.exs

```diff
@@ -0,0 +1,85 @@
+defmodule Web.Live.Resources.IndexTest do
+  use Web.ConnCase, async: true
+
+  setup do
+    account = Fixtures.Accounts.create_account()
+    identity = Fixtures.Auth.create_identity(account: account, actor: [type: :account_admin_user])
+
+    %{
+      account: account,
+      identity: identity
+    }
+  end
+
+  test "redirects to sign in page for unauthorized user", %{account: account, conn: conn} do
+    assert live(conn, ~p"/#{account}/resources") ==
+             {:error,
+              {:redirect,
+               %{
+                 to: ~p"/#{account}/sign_in",
+                 flash: %{"error" => "You must log in to access this page."}
+               }}}
+  end
+
+  test "renders breadcrumbs item", %{
+    account: account,
+    identity: identity,
+    conn: conn
+  } do
+    {:ok, _lv, html} =
+      conn
+      |> authorize_conn(identity)
+      |> live(~p"/#{account}/resources")
+
+    assert item = Floki.find(html, "[aria-label='Breadcrumb']")
+    breadcrumbs = String.trim(Floki.text(item))
+    assert breadcrumbs =~ "Resources"
+  end
+
+  test "renders add resource button", %{
+    account: account,
+    identity: identity,
+    conn: conn
+  } do
+    {:ok, _lv, html} =
+      conn
+      |> authorize_conn(identity)
+      |> live(~p"/#{account}/resources")
+
+    assert button = Floki.find(html, "a[href='/#{account.id}/resources/new']")
+    assert Floki.text(button) =~ "Add Resource"
+  end
+
+  test "renders resources table", %{
+    account: account,
+    identity: identity,
+    conn: conn
+  } do
+    group = Fixtures.Gateways.create_group(account: account)
+
+    resource =
+      Fixtures.Resources.create_resource(
+        account: account,
+        connections: [%{gateway_group_id: group.id}]
+      )
+
+    {:ok, lv, _html} =
+      conn
+      |> authorize_conn(identity)
+      |> live(~p"/#{account}/resources")
+
+    resource_rows =
+      lv
+      |> element("#resources")
+      |> render()
+      |> table_to_map()
+
+    Enum.each(resource_rows, fn row ->
+      assert row["name"] =~ resource.name
+      assert row["address"] =~ resource.address
+      assert row["gateway instance group"] =~ group.name_prefix
+    end)
+
+    # TODO: add assertion for Authorized Groups column
```

Comment:
@bmanifold did you plan to finish this TODO as part of PR?

Reply:
I had actually forgotten about this, because at the time of working on this PR the Policy PR hadn't been merged so it wasn't possible.  Since this PR was meant to just be for the New/Edit pages, do you mind if I cover this in a follow on PR?

Reply:
I think if it's small enough lets just add the assertions, or groups are not rendered too?

Reply:
Yeah, the groups on the index page aren't being rendered.  They are just static with `TODO` in the field right now.  That's the only reason I wanted to try and separate the work out.  If it were just the assertions I'd definitely put them in.

Reply:
If you feel like it we can do it in a separate PR after this one.

Reply:
👍 sounds good.  I always like keeping PRs smaller if possible.